src/pow.cpp

```diff
@@ -183,6 +262,40 @@ unsigned int CalculateNextWorkRequired(const CBlockIndex* pindexLast, const Cons
     return bnNew.GetCompact();
 }
 
+uint32_t CalculateNextWorkRequiredAux(const CBlockIndex* pindexLast, const Consensus::Params& params)
+{
+    arith_uint256 bnPowLimit = UintToArith256(params.aux_pow_limit);
+
+    // The very first block to have an auxiliary proof-of-work starts with a
+    // minimal difficulty.
```

Comment:
I am probably confused now, but I thought the aux pow was the "bitcoin" pow. Is the aux pow target separate from the aux chain native target (and then independently grow to the same value over time)? Or am I getting things mixed up?

Reply:
The auxiliary proof-of-work **is** the bitcoin header, yes, but the `nBits`-specified target value is specific to our chain. After all, not all of the bitcoin miners would be merge-mining at any given time.  So the bitcoin header would have some bitcoin-specified difficulty (stored in `AuxProofOfWork::m_aux_bits`), but this has no consensus value to us. All we care is that the bitcoin header meet **our** target, which is stored in `AuxProofOfWork::m_commit_bits`.

Reply:
I'll take your word for it. In my mind, the bitcoin header will always meet our target, unless our target exceeds the bitcoin nBits value, which requires >100% miners to merge mine tradecraft, but I'll probably understand this better after more review.

Reply:
The bitcoin header doesn't have to meet the bitcoin nBits target value. It just has to meet ours, which might be lower. So the "bitcoin header" which we include in our auxiliary proof of work might not actually be from from a valid bitcoin header.